web/guac/views.py

```diff
@@ -31,8 +31,8 @@ def index(request, task_id, session_data):
         if state[0] == 1:
             vmXml = dom.XMLDesc(0)
             root = ET.fromstring(vmXml)
-            graphics = root.find("./devices/graphics")
-            vncport = graphics.get("port")
+            graphics = root.find("./devices/graphics[@type=\"vnc\"]")
+            vncport = graphics.get("port") if graphics is not None else None
```

Comment:
if graphics else None, i can't remove that by myself as i don't have rights

`if X` does check if is not None/Bool